test.yml

```diff
@@ -7,17 +7,27 @@
 
   roles:
     - role: role_under_test
+      when: ansible_distribution_release == 'xenial' or (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7')
 
     - role: role_under_test
+      openio_repository_openstack_release: queens
       openio_repository_products:
         sds:
-          release: "16.10"
-      when: ansible_distribution != 'Debian'
+          release: '18.04'
+      when: ansible_distribution_release == 'bionic'
 
+
+##    - role: role_under_test
+##      openio_repository_products:
+##        sds:
+##          release: "16.10"
+##      when: ansible_distribution != 'Debian'
+#
     - role: role_under_test
+      openio_repository_mirror_host: mirror2.openio.io
```

Comment:
Why is this only on this role inclusion and not the other ones above ? Do we want to test both mirrors (a comment saying so would also be welcome, then) ?